### PR TITLE
419 - Add Autofit to Some Components

### PR DESCRIPTION
**Pull request by vivz753**
**Explain the details for making this change. What existing problem does the pull request solve?**

TODO: change base branch to main 
* merge in PR for #414 first

**Related github/jira issue (required)**:

**Steps necessary to review your pull request (required)**:

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [ ] A note to the change log.

### CODE REVIEW

**Comment by tmcconechy** [comment]
I put a WIP on since you want to wait for the other PR first 👍🏻 
When you get back to this can you please add steps to test as im not sure what to look at 100% here

File: demos/ids-data-grid/virtual-scroll.html [hunk 1]
```diff
@@ -1,11 +1,9 @@
-{{> ../layouts/head.html }}
 <ids-layout-grid auto="true">
   <ids-text font-size="12" type="h1">Data Grid</ids-text>
 </ids-layout-grid>
 <ids-layout-grid auto="true">
   <ids-layout-grid-cell>
-    <ids-data-grid id="data-grid-1" virtual-scroll="true" alternate-row-shading="true" row-height="small">
+    <ids-data-grid id="data-grid-1" virtual-scroll="true" row-height="medium">
```
Comment (tmcconechy) [inline demos/ids-data-grid/virtual-scroll.html:6]: Can we leave alternate-row on?

File: demos/ids-list-view/index.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.parent-container {
+  height: 800px;
+}
+
+.border {
+  border: 1px solid gray;
```
Comment (tmcconechy) [inline demos/ids-list-view/index.scss:6]: Can we use a color in the color pallette? https://main-enterprise.demo.design.infor.com/components/colors/example-index.html

**Review by tmcconechy — CHANGES_REQUESTED** [review]
Made some simple comments on the code. Also something is wierd about this PR in that it by passed all the checks for some reason?


Screen Shot 2021-11-12 at 10 44 03 AM

Maybe try to sync with main and push will fix it when you come back to this after its related PR is merged and all is updated..

**Comment by tmcconechy** [comment]
@vivz753 i tried to merge main on and something is still wrong with this PR. Can you try to reraise on a new branch and close this for now? Its wierd it did not run any checks. I've never seen this before...

**Comment by jmacaluso711** [comment]
@tmcconechy looks like the base branch is not `414-virtualscroll-dynamicheight` not `main` which is probably why there are no checks for the merge

**Comment by tmcconechy** [comment]
Closing for now to prevent merge. I think maybe re make a new branch and add the changes in and PR and see if that works?

**Comment by tmcconechy** [comment]
Good catch @jmacaluso711 I was pretty confused. Yeah @vivz753 will have to re branch from main and PR from that 👍🏻

**Pull request closed without merging**